### ISSUE TITLE
Url not formed correctly when used in browser

### ISSUE THREAD

**Issue by ThomasAuriel** [brianjgeiger/markdown-it-video#23]
By using markdown-it-video in *a browser*, the page can't find the content due to the shape of the url

src="//www.youtube.com/embed/dQw4w9WgXcQ"
should be
src="**https:**//www.youtube.com/embed/dQw4w9WgXcQ"

**Comment by brianjgeiger** [comment]
Hi, @ThomasAuriel. Can you give some more info? The primary use case for this plugin is in a browser, and I haven't seen that yet.

**Comment by ThomasAuriel** [comment]
Hi,

Yes of course.
So I use it through [browserify](http://browserify.org/).

I install it with the classic way:
``` sh
npm install --save markdown-it-npm
```
Once this is done, I "compile" my code (*main.js*) using browserify and the command:
``` sh
browserify -r jquery \
                  -r markdown-it \
                  -r markdown-it-checkbox \
                  -r markdown-it-imsize \
                  -r markdown-it-highlightjs \
                  -r markdown-it-katex \
                  -r markdown-it-math \
                  -r ascii2mathml \
                  -r babelify \
                  -r markdown-it-video \
                  /browserify/main.js -o /browserify/bundle.js
```

So the *bundle.js* embedded all the requirements in one file.

Other plugins I included work as a charm (excepted for markdown-it-math).
So when I want to debug, I saw that the library does not provide the **https:** at the begging of the url as describe on the first post:

*This does not work*
``` html
<iframe class="embed-responsive-item" id="youtubeplayer" type="text/html" src="//www.youtube.com/embed/dQw4w9WgXcQ" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen="" width="640" height="390" frameborder="0"></iframe>
```

*This works fine*
``` html
<iframe class="embed-responsive-item" id="youtubeplayer" type="text/html" src="https://www.youtube.com/embed/dQw4w9WgXcQ" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen="" width="640" height="390" frameborder="0"></iframe>
```

**Comment by brianjgeiger** [comment]
Closing this in favor of #25
